### ISSUE TITLE
validation datasets ？

### ISSUE THREAD

**Issue by liangsiping2019** [kangben258/MCITrack#21]
Hi author, why isn't a validation set set up in the training configuration for fine-tuning validation?
